py3status/modules/keyboard_locks.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+"""
+Monitor CapsLock, NumLock, and ScrLock keys
+
+Configuration parameters:
+    cache_timeout: refresh interval for this module (default 1)
+    format: display format for this module (default '{caps} {num} {scr}')
+    icon_capslock_off: show when Caps Lock is off (default 'CAP')
+    icon_capslock_on: show when Caps Lock is on (default 'CAP')
+    icon_numlock_off: show when Num Lock is off (default 'NUM')
+    icon_numlock_on: show when Num Lock is off (default 'NUM')
+    icon_scrlock_off: show when Scroll Lock is off (default 'SCR')
+    icon_scrlock_on: show when Scroll Lock is on (default 'SCR')
+
+Color options:
+    color_good: Lock on
+    color_bad: Lock off
+
+@author lasers
+"""
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+    cache_timeout = 1
+    format = '{caps} {num} {scr}'
+    icon_capslock_off = "CAP"
+    icon_capslock_on = "CAP"
+    icon_numlock_off = "NUM"
+    icon_numlock_on = "NUM"
+    icon_scrlock_off = "SCR"
+    icon_scrlock_on = "SCR"
+
+    def keyboard_lock(self):
+        out = self.py3.command_output('xset -q')
+
+        capslock_color = self.py3.COLOR_BAD
+        capslock_icon = self.icon_capslock_off
+        numlock_color = self.py3.COLOR_BAD
+        numlock_icon = self.icon_numlock_off
+        scrlock_color = self.py3.COLOR_BAD
+        scrlock_icon = self.icon_scrlock_off
+
+        if 'on' in out.split("Caps Lock:")[1][0:6]:
+            capslock_color = self.py3.COLOR_GOOD
+            capslock_icon = self.icon_capslock_on
```

Comment:
I had `from distutils.util import strtobool` to turn `bool(out.split("Caps Lock:")[1][0:6]:).strip())` into `True`/`False` to make it like... `if capslock:`, `if numlock:`, etc... but I got rid of that when optimizing because `if `on` in... ` will do just fine. 

I did what you did above earlier, but optimized to remove few lines. No `else`, no lines. Ha. It's back now tho. Fixed placeholders too. Agreed w/ `{scr}` to keep it simple.